src/tests/topp/ConsensusID_7_output.idXML

```diff
@@ -0,0 +1,95 @@
+			<UserParam type="string" name="Mascot:variable_modifications" value="Oxidation (M)"/>
+			<UserParam type="int" name="Mascot:missed_cleavages" value="1"/>
+			<UserParam type="float" name="Mascot:fragment_mass_tolerance" value="0.5"/>
+			<UserParam type="int" name="Mascot:fragment_mass_tolerance_ppm" value="0"/>
+			<UserParam type="float" name="Mascot:precursor_mass_tolerance" value="10.0"/>
+			<UserParam type="int" name="Mascot:precursor_mass_tolerance_ppm" value="0"/>
+			<UserParam type="string" name="Mascot:digestion_enzyme" value="Trypsin"/>
+			<UserParam type="string" name="SE:OMSSA" value="2.1.4"/>
+			<UserParam type="string" name="OMSSA:db" value="/nfs/wsi/bs/share/usr/nahnsen/sequences/Ecoli_K12_TaxID_83333.proteomes.decoy.fasta.psq"/>
+			<UserParam type="string" name="OMSSA:db_version" value=""/>
+			<UserParam type="string" name="OMSSA:taxonomy" value="0"/>
+			<UserParam type="string" name="OMSSA:charges" value="+1-+3"/>
+			<UserParam type="string" name="OMSSA:fixed_modifications" value=""/>
+			<UserParam type="string" name="OMSSA:variable_modifications" value="Oxidation (M)"/>
+			<UserParam type="int" name="OMSSA:missed_cleavages" value="1"/>
+			<UserParam type="float" name="OMSSA:fragment_mass_tolerance" value="0.5"/>
+			<UserParam type="int" name="OMSSA:fragment_mass_tolerance_ppm" value="0"/>
+			<UserParam type="float" name="OMSSA:precursor_mass_tolerance" value="0.01"/>
+			<UserParam type="int" name="OMSSA:precursor_mass_tolerance_ppm" value="0"/>
+			<UserParam type="string" name="OMSSA:digestion_enzyme" value="Trypsin"/>
+			<UserParam type="string" name="SE:XTandem" value=""/>
+			<UserParam type="string" name="XTandem:db" value="/share/usr/nahnsen/sequences/Ecoli_K12_TaxID_83333.proteomes.decoy.fasta.pro"/>
+			<UserParam type="string" name="XTandem:db_version" value=""/>
+			<UserParam type="string" name="XTandem:taxonomy" value=""/>
+			<UserParam type="string" name="XTandem:charges" value="+1-+3"/>
+			<UserParam type="string" name="XTandem:fixed_modifications" value=""/>
+			<UserParam type="string" name="XTandem:variable_modifications" value="Oxidation (M)"/>
+			<UserParam type="int" name="XTandem:missed_cleavages" value="1"/>
+			<UserParam type="float" name="XTandem:fragment_mass_tolerance" value="0.5"/>
+			<UserParam type="int" name="XTandem:fragment_mass_tolerance_ppm" value="0"/>
+			<UserParam type="float" name="XTandem:precursor_mass_tolerance" value="10.0"/>
+			<UserParam type="int" name="XTandem:precursor_mass_tolerance_ppm" value="0"/>
+			<UserParam type="string" name="XTandem:digestion_enzyme" value="unknown_enzyme"/>
+		</ProteinIdentification>
+		<PeptideIdentification score_type="Posterior Error Probability" higher_score_better="false" significance_threshold="0.0" MZ="695.364599999999996" RT="2233.105000000000018" spectrum_reference="index=1" >
+			<PeptideHit score="0.54" sequence="QRESTATDILQK" charge="2" >
+				<UserParam type="float" name="consensus_support" value="1.0"/>
```

Comment:
I think it would be cool to have the option to list the old scores as meta values

Reply:
Unfortunately, there is no info about the search engine in the PeptideIdentifications.

Reply:
looks good to me now